### ISSUE TITLE
Unconference event

### ISSUE THREAD

**Issue by sunnivin** [nordic-rse/conference-contributions#18]
### It is a:
- [ ] Short talk (10 minutes including discussion)
- [x] Longer talk (20 minutes including discussion)
- [ ] Discussion (up to 70 minutes)
- [ ] Demonstration (up to 70 minutes)
- [ ] Workshop (up to 70 minutes)
- [ ] ReproHack (up to 70 minutes)
- [ ] Something else (up to 70 minutes)

### Do you need help?
- [ ] I would like someone to help me plan the contribution

### Would you be comfortable if your event was recorded?
- [x] Yes
- [ ] No

### Please indicate on which day(s) the contribution can happen (tick both if you don't mind which one):
- [x] October 18, except from 13.30-14.30
- [x] October 19, except from 11.00-14.30

### Give a short description of your contribution:
Title: Make model validation sexy again 

Code written to simulate real physical systems typically needs to provide an ensemble of input parameters to create a valid simulation. Writing logic for model validation is a repetitive and tedious task that needs to be handled with great care when performed manually. 

This talk will show how the modern software stack can simplify model validation of a traditional finite element method. With the combination of pydantic, docker, and Fast-API we can even make Fortran77 sexy again.

**Comment by lucaferranti** [comment]
Hi @sunnivin :wave:,

thank you for your submission and apologies for the slow follow-up.

We are approaching the unconference and I would like to share some information and ask a few things

1. The schedule is now on the webpage: https://nordic-rse.org/events/2022-online-unconference/#how-it-works It will be on 18th October and 19th October, **are you still willling to participate?**
2. Would you like to deliver your talk on the first unconference slot of the second day, that is **October 19th 13:10 CET**?
3. Are you ok with the title and abstract being published on the webpage?
4. If you haven't already, make sure to [register for the unconference](https://indico.neic.no/event/232/)

Thank you for your contribution and if you have any questions, feel free to ping me

/Luca

**Comment by sunnivin** [comment]
> The schedule is now on the webpage: https://nordic-rse.org/events/2022-online-unconference/#how-it-works It will be on 18th October and 19th October, are you still willling to participate?

O yes, looking very much forward to join in :) 

> Would you like to deliver your talk on the first unconference slot of the second day, that is October 19th 13:10 CET?

I'm available for the Unconference session 1 at 14:50 on Tuesday or the Unconference session 3 at 14:40 on Wednesday. I hope my contribution fits in at these time intervals.

> Are you ok with the title and abstract being published on the webpage?

Yes! 

> If you haven't already, make sure to [register for the unconference](https://indico.neic.no/event/232/)

I'm signed up :)

**Comment by lucaferranti** [comment]
> I'm available for the Unconference session 1 at 14:50 on Tuesday or the Unconference session 3 at 14:40 on Wednesday. I hope my contribution fits in at these time intervals.

yes no problem! Do you have any preferences between the two time slots?

**Comment by sunnivin** [comment]
> yes no problem! Do you have any preferences between the two time slots?

If I can choose I would like the Wednesday afternoon session :)

**Comment by lucaferranti** [comment]
that's perfect! I'll add the confirmed sessions today to the webpage
